lib/protractor-testability-plugin.js

```diff
@@ -74,15 +78,41 @@ return {
                 };
             }
 
+            function patchPromiseFunction(set) {
+                var setFn = window[set];
+
+                window[set] = function () {
+                    var ref;
+
+                    ref = setFn.apply(window, arguments);
+
+                    ref.then(function (result) {
+                        window.testability.wait.oneLess();
+                        return result;
+                    });
+
+                    window.testability.wait.oneMore();
+
+                    return ref;
+                };
+            }
+
             patchFunction('setTimeout', 'clearTimeout', true);
             patchFunction('setInterval', 'clearInterval', true);
             patchFunction('setImmediate', 'clearImmediate', false);
+
+            patchPromiseFunction('fetch');
```

Comment:
The same as the other one.... Seems to have value independently of the `readyFor`. A separate PR for this will be perfect :-) if it doesn't take you too much time to perform.